docs/octavia-ingress-controller/using-octavia-ingress-controller.md

```diff
@@ -449,3 +450,33 @@ Ingress and enable the more secure HTTPS protocol.
 > NOTE: octavia-ingress-controller currently doesn't support to integrate with
 > `cert-manager` to create the non-existing secret dynamically. Could be improved
 > in the future.
+
+## Allow CIDRs
+
+You can use the key `octavia.ingress.kubernetes.io/whitelist-source-range` annotations to whitelist CIDRs.
```

Comment:
Personally, I think the description in the issue page looks more user friendly than here.

Reply:
Fixed by https://github.com/kubernetes/cloud-provider-openstack/pull/1704/commits/06acf15e30d6bd467d32ff13c4fbbd30ecc9b439.